workflow/get_model_observed_tags.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env pythona
+import argparse
+import sys
+import subprocess
+import threading
+from queue import Queue
+import traceback
+from asapp.common import log, cli
+from paths import *
+
+
+class ModelServer:
+    def __init__(self, release, baseline, model):
+        self._release = release
+        self._baseline = baseline
+        self._model = model
+
+
+    def get_model_from_s3(self):
+        subprocess.call(['model_stash', '--bucket', 'asapp-models-dev','get', self._model])
+
+
+    def start_server(self, queue):
+        server = subprocess.Popen(['pythona',
```

Comment:
Any reason for using `Popen` here? I believe `subprocess.run` wraps `Popen`, and it looks like you used it successfully just below.